package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "lab-express-basic-site",
+  "version": "1.0.0",
+  "description": "![logo_ironhack_blue 7](https://user-images.githubusercontent.com/23629340/40541063-a07a0a8a-601a-11e8-91b5-2f13e4e6b441.png)",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Aquí podriamos agregar el script para usar el comando en nuestra terminal de manera practica y nodemon para usarlo en etapa de producción este se ejecuta con run= "npm run dev" 😉

```suggestion
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "node app.js",
    "dev": "nodemon app.js"
```